blog/__init__.py

```diff
@@ -0,0 +1 @@
+"""This package initializes the Blog application."""
```

Comment:
No, packages don't initialize anything. `import blog` isn't supposed to have side effects. They should just contain/group stuff.

Reply:
Marking as resolved.